src/main/java/uk/gov/hmcts/darts/dailylist/entity/DailyListEntity.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.darts.dailylist.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.OffsetDateTime;
+import java.util.Date;
+
+@Entity
```

Comment:
Should we not be updating this Entity?
https://github.com/hmcts/darts-api/blob/DMP-180_Create_New_Daily_List_table/src/main/java/uk/gov/hmcts/darts/common/entity/DailyList.java

I think we would need to add the moj_dal_seq SEQUENCE similar to 
https://github.com/hmcts/darts-api/blob/DMP-180_Create_New_Daily_List_table/src/main/java/uk/gov/hmcts/darts/common/entity/Media.java#L22-L23